scripts/tempColorTokenBuild.ts

```diff
@@ -193,6 +199,6 @@ export const buildDesignTokens = (buildOptions: ConfigGeneratorOptions): void =>
 }
 
 buildDesignTokens({
-  buildPath: 'tokens-v2-private/',
+  buildPath: 'tokens-v3-private/',
```

Comment:
So this means that we're no longer supporting / issuing future updates to tokens in `tokens-v2-private/`. Wondering if we need to communicate that somewhere 🤔

Reply:
No, because the script we use for this is unchanged. So we can still issue updates.

Reply:
This is the build script for tokens that are currently shipped https://github.com/primer/primitives/blob/main/build.js